rclcpp_lifecycle/include/rclcpp_lifecycle/lifecycle_node.hpp

```diff
@@ -128,6 +131,22 @@ class LifecycleNode : public node_interfaces::LifecycleNodeInterface,
   const std::vector<rclcpp::callback_group::CallbackGroup::WeakPtr> &
   get_callback_groups() const;
 
+  /// Create and return a Publisher.
+  /**
+   * \param[in] topic_name The topic for this publisher to publish on.
+   * \param[in] qos The Quality of Service settings for this publisher.
+   * \param[in] options The publisher options for this publisher.
+   * \return Shared pointer to the created lifecycle publisher.
+   */
+  template<typename MessageT, typename AllocatorT = std::allocator<void>>
+  std::shared_ptr<rclcpp_lifecycle::LifecyclePublisher<MessageT, AllocatorT>>
+  create_publisher(
+    const std::string & topic_name,
+    const rclcpp::QoS & qos,
+    const rclcpp::PublisherOptionsWithAllocator<AllocatorT> & options = (
+      rclcpp::PublisherOptionsWithAllocator<AllocatorT>()
+    ));
```

Comment:
Maybe. is better to apply https://github.com/ros2/rclcpp/commit/7c59d9045b6a40a24d6bb0d7e0f46dbd8e0b3f5e style change here too. If in this case windows is happy, I don't mind.

Reply:
Oh, that's an accident. I'll have to add back the extra `()` to keep uncrustify happy, but for the namespace, I would prefer to use them where I can, it's just using them in the node.hpp, for what ever reason, causes the compiler error...

Reply:
😭 I have to remove the namespace ___AND___ the extra parenthesis to avoid the error... It's the same error with one or the other or both... wtf Windows.

Reply:
At least in `node.hpp` I do, here I think the `rclcpp::` is required, since it's from a different namespace.